src/feedback.js

```diff
@@ -13,6 +13,7 @@ export default class Feedback {
 
 		const defaultOptions = {
 			id: 'feedback',
+			emailplaceholder: 'Email (optional)',
```

Comment:
To keep the naming consistent I would prefer if you used [camelCase](https://techterms.com/definition/camelcase) here as well and named the new option `emailPlaceholder`.